test/browser/features/network_breadcrumbs.feature

```diff
@@ -0,0 +1,12 @@
+@network_breadcrumbs
+Feature: Network breadcrumbs
+
+Scenario: fetch
+  When I navigate to the test URL "/network_breadcrumbs/script/fetch.html"
+  And I wait to receive an error
+  Then the error is a valid browser payload for the error reporting API
```

Comment:
should there be some way to assert that the "valid browser payload" actually contained a network breadcrumb or two? Or is that the kind of thing we do at unit test level only?

Reply:
I've added some tests to validate a successful and failure for both fetch and xmlhttprequest breadcrumbs 👍🏻